piptools/repositories/base.py

```diff
@@ -61,3 +62,8 @@ def session(self) -> PipSession:
     @abstractmethod
     def finder(self) -> PackageFinder:
         """Returns a package finder to interact with simple repository API (PEP 503)"""
+
+    @property
+    @abstractmethod
+    def command(self) -> InstallCommand:
+        """Returns an install command"""
```

Comment:
Let's use a PEP 257 compliant docstring.
```suggestion
        """Return an install command."""
```

Reply:
Addressed in 81419f9f1090bda0761020e7e9832ba279a5fd4d